server/osc_dash.py

```
@@ -0,0 +1,402 @@
+        per_series.setdefault(k, []).append(r)
+    hist_max=[] # buckets 0-50c in 5c steps
+    buckets = list(range(0,55,5))
+    hist = {b:0 for b in buckets}
+    for r in rows:
+        m = max(r.get("max_up",0), r.get("max_down",0))*100
+        for b in buckets:
+            if m < b+5:
+                hist[b]+=1
+                break
+    # start deviation histogram
+    hist_start={b:0 for b in [0,1,2,3,5,10]}
```

Ruff (0.16.2)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)